README.md

```diff
@@ -144,6 +144,29 @@ unzip ~/tmp_fmltc/closure-library_v20200406.zip
 popd
 ```
 
+## Install JDK
+Depending on your OS and distribution there are various ways to install JDK.
+
+- Ubuntu/Debian
+
+```
+sudo apt install openjdk-11-jdk #Ubuntu
+```
+- Manjaro
+
+```
+sudo pacman -S jre11-openjdk-headless jre11-openjdk jdk11-openjdk openjdk11-doc openjdk11-src
+```
+
+- Arch
+
+```
+sudo pacman -S jdk11-openjdk
+```
+- Other Linux distributions 
+
+https://openjdk.java.net/install/
+
```

Comment:
If we have Linux instructions should we have Windows and Mac instructions?

Reply:
Makes sense. I'll add in instructions for Mac and Windows. It would also make sense to add in a note about what OS's have been tested and how to do it on a given OS. So for Windows it would be using Git Bash or WSL, and for Mac and Linux it would be just be using regular terminal.

Reply:
I have a MacBook Pro, but I use a Linux box for development.